src/app/api/v1/fxa-rp-events/route.ts

```diff
@@ -310,15 +310,19 @@ export async function POST(request: NextRequest) {
               });
 
               captureException(
-                new Error(`No OneRep profile Id found, subscriber: ${
+                new Error(`User subscribed but no OneRep profile Id found, user: ${
```

Comment:
I know you didn't introduce it, but it seems odd to construct an `Error` just to pass to `captureException`... I wonder if `captureMessage` is a better option here.

Reply:
Good call

Reply:
Fixed a bunch of them